crystalball.gemspec

```diff
@@ -31,6 +31,7 @@ Gem::Specification.new do |spec|
   spec.require_paths = ["lib"]
 
   spec.add_dependency 'git'
+  spec.add_dependency 'parser'
```

Comment:
I think this shouldn't be a default dependency. Let's move it to development dependency only.
We need to mention in README that if you want to use parser strategy you need to add `parser` to your Gemfile.